editorextensions/codebeamer-custom-ui/modal/src/pages/import/components/query/Query.tsx

```diff
@@ -13,21 +14,26 @@ export default function Query() {
 		(state: RootState) => state.userSettings
 	);
 
-	if (!advancedSearch) {
-		return (
-			<div className="grid fade-in-quick">
-				<div className="cs1 ce3">
-					<TrackerSelect />
+	return (
+		<div className="flex-row align-items-center">
+			{advancedSearch ? (
+				<div className="grid fade-in-quick">
+					<div className="cs1 ce3">
+						<TrackerSelect />
+					</div>
+					<div className="cs4 ce7 text-center form-group">
+						<FilterInput />
+					</div>
+					<div className="cs8 ce12 text-center active-filters-container">
+						<ActiveFilters />
+					</div>
 				</div>
-				<div className="cs4 ce7 text-center">
-					<FilterInput />
+			) : (
+				<div className="w-100">
+					<CbqlInput />
 				</div>
-				<div className="cs8 ce12 text-center active-filters-container">
-					<ActiveFilters />
-				</div>
-			</div>
-		);
-	} else {
-		return <CbqlInput />;
-	}
+			)}
+			<CbqlToggle />
+		</div>
```

Comment:
A few highlights on the CBQLToggle button:
![cb_lucid_ui_stains](https://github.com/codeBeamer-Extensions-and-Addons/codebeamer-lucidspark/assets/70019423/a9366461-436c-42e8-9c29-b0b7fbdcc28a)

I imagine you're aware, but the button changing width with state and its unaligned vertical positioning are a notable stain that we should get rid of, if we're _refining_ the UI in the first place.

Make sure the button dimensions stay consistent, and reconsider the button's placement.  

A single `rem` of `margin-top` happens to be rather nicely centered with the input:
![image](https://github.com/codeBeamer-Extensions-and-Addons/codebeamer-lucidspark/assets/70019423/d4925e17-a678-4b45-9285-b9ec9e8e6cc5)

And to distinguish this button from the rest of the input form, I would think something like a border and some padding like this would be adequate:
![image](https://github.com/codeBeamer-Extensions-and-Addons/codebeamer-lucidspark/assets/70019423/fd674b7d-544c-43fa-9e15-d81a70573efd)  
That way, the button doesn't look completely out of place / random.